intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -428,29 +432,32 @@ \subsection{Generic resource requirements}
 \end{prompt}
 
 For the simplest cases, only the amount of maximum estimated execution time
-(called ``walltime'') is really important. Here, the job will not require more
-than 2 hours, 30 minutes to complete. As soon as the job would take more time,
+(called ``walltime'') is really important. Here, the job requests a reservation of
+2 hours, 30 minutes. As soon as the job starts to take more time,
 it will be ``killed'' (terminated) by the job scheduler.  There is absolutly no
 harm if you \emph{slightly} overestimate the maximum execution time. \\
 
 \begin{prompt}
 %\shellcmd{qsub -l mem=4gb}%
 \end{prompt}
 
-The job requires no more than 4 GB of memory. \\
+The job requests a reservation of 4 GB of RAM memory. As soon as the job tries to use more memory,
+it will be ``killed'' (terminated) by the job scheduler.  There is absolutly no
```

Comment:
`absolutely`

Maybe this is overstated a bit, since it may have a negative effect on queuing time (requested more resources may mean that the job has the wait longer).

Reply:
ok.